translations/en.json

```diff
@@ -491,6 +491,67 @@
 				"open-positions": "Open Positions"
 			}
 		},
+		"stake": {
+			"portfolio": {
+				"title": "Portfolio",
```

Comment:
Might consider renaming this to "Staking", "Staking Portfolio" or "Staking Dashboard" or "Staking Overview" or "KWENTA Staking" or "$Kwenta Portfolio" or so, the term _Portfolio_ alone would be confusing to me here given we already have "Portfolio" on the Dashboard Overview. Also it's not immediately clear *what* we're staking here :D

Prob best to discuss with some English-native CC :)

Reply:
It has been renamed.